### PR TITLE
Sass dependency

### PR DESCRIPTION
**Pull request by G-Joe**
This solves issue #355 by moving sass from dependency to dev dependency

## Summary by CodeRabbit

- **Chores**
  - Updated pre-commit hook to skip linting.
  - Added `sass` dependency to the project.

- **Refactor**
  - Converted `ColorBubble` and `ColorsTemplate` components from arrow functions to function declarations for clarity.

### CODE REVIEW

File: src/examples/Colors/ColorsTemplate.js [hunk 1]
```diff
@@ -7,437 +7,77 @@ const ColorBubble = ({colorClass}) => {
     );
```
Comment (kotAPI) [inline src/examples/Colors/ColorsTemplate.js:7]: can you revert these?

Comment (G-Joe) [reply src/examples/Colors/ColorsTemplate.js]: Done

Comment (G-Joe) [reply src/examples/Colors/ColorsTemplate.js]: Is there a challenge here or you meant something else

**Review by kotAPI — APPROVED** [review]

**Comment by kotAPI** [comment]
You can fix the merge conflicts and we're good

**Pull request merged**